alibi/datasets/default.py

```diff
@@ -325,9 +324,9 @@ def fetch_adult(features_drop: Optional[list] = None, return_X_y: bool = False,
 
     # only return data values
     data = data.values
-    target_names = ['<=50K', '>50K']
-
     if return_X_y:
         return data, labels
 
+    target_names = ['<=50K', '>50K']
+
```

Comment:
What's the logic behind moving this here? It makes more sense defining `target_names` next to `data` to me...

Reply:
I believe it's so that we don't define it needlessly if the function returns early, though it's a minor point.